src/commands/dev/reload.ts

```diff
@@ -0,0 +1,85 @@
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { codeBlock } from '@discordjs/builders';
+import { CommandInteraction } from 'discord.js';
+import { execSync } from 'child_process';
+import ApplicationCommandHandler from '@handlers/ApplicationCommandHandler';
+import ComponentHandler from '@handlers/ComponentHandler';
+
+export async function run(client: FluorineClient, interaction: CommandInteraction) {
+    await interaction.deferReply({ ephemeral: true, fetchReply: true });
+    const type = interaction.options.getString('type');
+    const module = interaction.options.getString('module');
+
+    try {
+        execSync('npm run build');
```

Comment:
```suggestion
import ApplicationCommandHandler from '@handlers/ApplicationCommandHandler';
import ComponentHandler from '@handlers/ComponentHandler';

export async function run(client: FluorineClient, interaction: CommandInteraction) {
    await interaction.deferReply({ ephemeral: true, fetchReply: true });
    const type = interaction.options.getString('type');
    const module = interaction.options.getString('module');

    try {
```
it already builds on /update

Reply:
guess ill add a check to see if we're on dev